### ISSUE TITLE
video link that is not youtube does not work

### ISSUE THREAD

**Issue by josefspr** [daat-hamakom/maps#137]
see event: Handwritten version of Hatikvah by Naftali Herz Imber
https://daat-hamakom-data.herokuapp.com/admin/maps/event/496/change/

**Comment by yuvadm** [comment]
@josefspr we never discussed supporting arbitrary links. ~~Either way the widget located at http://cdnapi.kaltura.com/index.php/extwidget/openGraph/wid/1_9efw9gmn doesn't work for me at all~~

**Comment by josefspr** [comment]
ok, to wish list... would it be possible to have embedded code inside descriptions ?

**Comment by yuvadm** [comment]
@josefspr yes, if you edit the description HTML it will propagate all the way up to the interface, so you can technically throw in arbitrary embeds there. No promises on how they will render in terms of width/height unless you can control that via the embed params.
